Assets/MRTK/Core/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -531,27 +531,44 @@ public static void DrawSubProfileEditor(Object profileObject, bool renderProfile
             EditorGUILayout.EndVertical();
         }
 
+        public static bool DrawProfileDropDownList(SerializedProperty property, BaseMixedRealityProfile profile, Object oldProfileObject, Type profileType, bool requiresProfile, bool showAddButton)
+        {
+            return DrawProfileDropDownList(property, profile, oldProfileObject, new Type[] { profileType }, requiresProfile, showAddButton);
+        }
+
         /// <summary>
         /// Draws a dropdown with all available profiles of profilyType.
         /// </summary>
         /// <returns>True if property was changed.</returns>
-        public static bool DrawProfileDropDownList(SerializedProperty property, BaseMixedRealityProfile profile, Object oldProfileObject, Type profileType, bool showAddButton)
```

Comment:
Does this old method signature still exist? We should re-add it if not and mark it as obsolete

Reply:
Yeah, the function still signature still exists, it's a wrapper for this function now. (line 534)